lumigator/frontend/src/components/molecules/LPageHeader.vue

```diff
@@ -5,7 +5,8 @@
   >
     <div class="l-page-header__text-content">
       <h3>{{ title }} </h3>
-      <p>{{ subtitle }}</p>
+      <p class="l-page-header__text-content-description">{{ description }}</p>
+      <!-- <p>{{ subtitle }}</p> -->
```

Comment:
❓ can this comment go now, or does it need to hang around?

Reply:
It is best to keep it because this is a reusable component